src/gretel_trainer/models.py

```diff
@@ -145,27 +143,42 @@ class GretelCTGAN(_BaseConfig):
         config (str/dict, optional): Either a string representing the path to the config on the local filesystem, a string representing a path to the default Gretel configurations, or a dictionary containing the configurations. Default: "synthetics/high-dimensionality", a default Gretel configuration
         max_rows (int, optional): The number of rows of synthetic data to generate. Defaults to 50000
         max_header_clusters (int, optional): Default: 500
-        enable_privacy_filters (bool, optional): Default: False
+        enable_privacy_filters (bool, optional): This parameter is deprecated and will be removed in future versions.
     """
-    _max_header_clusters_limit: int = 1_000
+
+    _max_header_clusters_limit: int = 5_000
     _max_rows_limit: int = 5_000_000
     _model_slug: str = "ctgan"
 
     def __init__(
         self,
         config="synthetics/high-dimensionality",
-        max_rows=50_000,
-        max_header_clusters=500,
-        enable_privacy_filters=False,
+        max_rows=1_000_000,
+        max_header_clusters=1_000,
+        enable_privacy_filters=None,
```

Comment:
Is it worth adding a log warning about this? Something like:
```python
if enable_privacy_filters is not None:
    logger.warning("enable_privacy_filters is deprecated...")
```

Reply:
Yeah, that's a good idea!